src/pages/CommitDetailPage/subRoute/FilesChangedTab/FilesChangedTab.tsx

```diff
@@ -44,6 +46,9 @@ function FilesChanged() {
 
   return (
     <Suspense fallback={<Loader />}>
+      <div className="flex justify-end bg-ds-gray-primary p-2">
+        <ComponentsSelector />
+      </div>
```

Comment:
How to put this beside the flags selector? Currently it is one line below doing it like this